packages/ui/layouts/default.vue

```diff
@@ -60,14 +67,25 @@ html {
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
   box-sizing: border-box;
-  background: var(--bg);
   scroll-behavior: smooth;
+  background-color: var(--bg);
+}
+.background {
+  background-color: var(--bg);
+  transition: all 0.3s ease;
+  &.preload {
+    transition: none !important;
+    -webkit-transition: none !important;
+    -moz-transition: none !important;
+    -ms-transition: none !important;
+    -o-transition: none !important;
+  }
```

Comment:
I can't see any change even when I force `preload` to be always `true`

Reply:
The difference is when changing the theme  in the navbar background color